homeassistant/components/device_tracker/__init__.py

```diff
@@ -13,15 +13,20 @@
     ATTR_DEV_ID,
     ATTR_GPS,
     ATTR_HOST_NAME,
+    ATTR_IP,
     ATTR_LOCATION_NAME,
     ATTR_MAC,
     ATTR_SOURCE_TYPE,
```

Comment:
Should we expose `ATR_CONSIDER_HOME`, too? I believe that's the only one missing now.

Reply:
It can also be added later if necessary. Let's move forward with this PR as is.